tests.sh

```diff
@@ -169,187 +198,329 @@ tests_diff() {
     tests_inc_asserts_count
 }
 
-tests_diff_blank() {
-    tests_diff "$1" "$2" "-B"
+# @description Returns file containing stdout of last command.
+#
+# @example
+#   tests:eval echo 123
+#   cat $(tests:get-stdout) # will echo 123
+#
+# @echo Filename containing stdout.
+tests:get-stdout() {
```

Comment:
weird name, returns file with stdout instead of output

guess should create `tests:stdout()` for contents of stdout